desktop/windows/release-notes/index.md

```diff
@@ -41,6 +41,26 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 ### Bug fixes and minor changes
 
+- Fixed an issue with manual proxy settings which caused problems when pulling images. Fixes [docker/for-win#12714](https://github.com/docker/for-win/issues/12714).
```

Comment:
We also got a Mac ticket today. So we could link to 

```suggestion
- Fixed an issue with manual proxy settings which caused problems when pulling images. Fixes [docker/for-mac#6315](https://github.com/docker/for-mac/issues/6315).
```

Reply:
added this issue as well!